xarray/core/dataset.py

```diff
@@ -2741,6 +2741,56 @@ def roll(self, **shifts):
 
         return self._replace_vars_and_dims(variables)
 
+    def sortby(self, variables, ascending=True):
+        """
+        Sorts the dataset, either along specified dimensions,
```

Comment:
Can you please add a short one-line description at the top? That is the standard numpy docstring format.